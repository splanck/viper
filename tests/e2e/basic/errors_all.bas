ON ERROR GOTO H1
x = 1 : y = 0 : PRINT x \ y : PRINT "unreached"
H1:
  PRINT "dz" : RESUME NEXT
  PRINT "cont1"

ON ERROR GOTO H2
OPEN "missing.txt" FOR INPUT AS #1
H2:
  PRINT "fnf" : RESUME NEXT

ON ERROR GOTO H3
DIM a(1 TO 2) : PRINT a(3)
H3:
  PRINT "bounds" : RESUME NEXT
