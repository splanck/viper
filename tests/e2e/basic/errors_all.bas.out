dz
cont1
fnf
bounds
